src/frontend/handlers.go

```diff
@@ -116,7 +115,7 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		"platform_css":      plat.css,
 		"platform_name":     plat.provider,
 		"is_cymbal_brand":   isCymbalBrand,
-		"deploymentDetails": getDeploymentDetails(r),
+		"deploymentDetails": getDeploymentDetailsIfLoaded(),
```

Comment:
Minor nit: if the `getDeploymentDetailsIfLoaded()` function simply returns `deploymentDetailsMap` as-is (as it should), why don't we remove the function and replace this line with `"deploymentDetails": deploymentDetailsMap`?

If the map is filled up, then we will pass it to be rendered, and if the map hasn't been filled up yet, we'll just be passing on `nil` but that's okay since we're catching that case in the `{{ if $.deploymentDetails }}` of the render.

Reply:
Done. Just replaced `getDeploymentDetailsIfLoaded()` with `deploymentDetailsMap`.